dali/test/python/test_video_reader_resize.py

```diff
@@ -150,3 +150,7 @@ def test_video_resize(batch_size=2):
     for vp in video_reader_params:
         for rp in resize_params:
             yield run_for_params, batch_size, vp, rp
+
+if __name__ == "__main__":
```

Comment:
Bad idea. It quickly becomes obsolete. If you really must you can try to query this module for all available functions and run all that falls into the default nosetests regexp - https://nose.readthedocs.io/en/latest/usage.html#cmdoption-m

Reply:
It's here because nosetests launches new processes and defeats debugging attempts (and does other ugly things that have to be disabled with "no-smoke" and "don't-catch-fire" kind of flags).

Reply:
Then ` query this module for all available functions and run all that falls into the default nosetests regexp - https://nose.readthedocs.io/en/latest/usage.html#cmdoption-m` sounds like a way to do.

Reply:
working on it...

Reply:
Done.

Reply:
+1 for official API - but I'd still prefer to have a way to run the tests with fewer levels of indirections.
I think I agree that this solution is not good after all - while it catches all tests, it's not what I really want here. I'll remove it and revert the changes to test_utils.
Regarding the plugin - I think it doesn't launch GDB, but a python debugger (?).